presidio_evaluator/models/presidio_analyzer_wrapper.py

```diff
@@ -1,6 +1,7 @@
 from typing import List, Optional, Dict
```

Comment:
Changes here:
- improved the way kwargs are passed to the analyzer.analyze method, to allow users to pass parameters in an easier way
- updated the mapping to presidio's entities to be more comprehensive.